### PR TITLE
An attempt at flattening the detect example

### PR DESCRIPTION
**Pull request by gguuss**

### CODE REVIEW

File: vision/cloud-client/detect/detect.py [hunk 1]
```diff
@@ -393,31 +393,22 @@ def detect_document(path):
...
-
-            page_text = page_text + block_text
+            block_text = ''
+            for symbol in block_symbols:
+                block_text = block_text + symbol.text
 
-        print('Page Content:\n{}'.format(page_text))
-        print('Page Dimensions: w: {} h: {}'.format(page.width, page.height))
+            print('Block Content: {}'.format(block_text))
+            print('Block Bounds:\n {}'.format(block.bounding_box))
```
Comment (gguuss) [inline vision/cloud-client/detect/detect.py]: Would this be better if written similar to:
```
        vertices = (['({},{})'.format(bound.x_coordinate, bound.y_coordinate)
                    for bound in face.bounds.vertices])
```

Comment (jonparrott) [inline vision/cloud-client/detect/detect.py]: What does the repr look like right now?

Comment (gguuss) [inline vision/cloud-client/detect/detect.py]:
```
Block Content: Back
Block Bounds:
 vertices {
  x: 134
  y: 852
}
vertices {
  x: 191
  y: 857
}
vertices {
  x: 189
  y: 873
}
vertices {
  x: 132
  y: 868
}

Block Content: 37%
Block Bounds:
 vertices {
  x: 712
  y: 699
}
vertices {
  x: 758
  y: 702
}
vertices {
  x: 757
  y: 718
}
vertices {
  x: 711
  y: 715
}
```

Comment (jonparrott) [inline vision/cloud-client/detect/detect.py]: It's fine as-is.

**Pull request merged**